src/ElementQueries.js

```diff
@@ -180,30 +180,18 @@
             else allQueries[mode][property][value] += ','+selector;
         }
 
-        function getQuery() {
-            var query;
-            if (document.querySelectorAll) query = document.querySelectorAll.bind(document);
-            if (!query && 'undefined' !== typeof $$) query = $$;
-            if (!query && 'undefined' !== typeof jQuery) query = jQuery;
-
-            if (!query) {
-                throw 'No document.querySelectorAll, jQuery or Mootools\'s $$ found.';
-            }
-
-            return query;
-        }
```

Comment:
You should really not just delete that code. It removes features from the whole library and that change is not even related to the given pull request description.